subprojects/workers/src/main/java/org/gradle/workers/internal/DefaultWorkerExecutor.java

```diff
@@ -102,6 +104,20 @@ public DefaultWorkResult call() throws Exception {
         registerAsyncWork(spec.getDisplayName(), workerDaemonResult);
     }
 
+    private WorkerFactory getWorkerFactory(IsolationMode isolationMode) {
+        switch(isolationMode) {
+            case AUTO:
+            case CLASSLOADER:
+                return workerInProcessFactory;
```

Comment:
renamed to isolatedClassloaderWorkerFactory, noIsolationWorkerFactory, and daemonWorkerFactory.